src/components/we-are-using/index.js

```diff
@@ -247,6 +247,10 @@ const companies = [
 		name: 'Aarogya Setu',
 		href: 'https://www.aarogyasetu.gov.in/',
 		logo: 'aarogyasetu.svg'
+	},{
+		name: 'Extrieve Technologies',
+		href: 'https://www.extrieve.com/',
```

Comment:
Something is off with the SSL certificate:

![Screenshot 2020-06-25 at 14 21 21](https://user-images.githubusercontent.com/1062408/85718832-2c9ac180-b6ef-11ea-9987-1fae1b6c1071.png)

Reply:
Loads only via `http`